manifests/application/migrator/daemon.pp

```diff
@@ -41,14 +41,14 @@
     $jsvc_version = 'latest',
     $owner = 'root',
     $group = 'root',
-    $service_name,
+    $service_name = undef,
     $jsvc_exec = '$(which jsvc)',
     $java_home = undef,
-    $service_out_logfile,
-    $service_err_logfile,
+    $service_out_logfile = undef,
+    $service_err_logfile = undef,
     $lib_dir = undef,
-    $jar_path,
-    $mainClass,
+    $jar_path = undef,
+    $mainClass = undef,
```

Comment:
Please do not make these undef. Doing so makes them optional, and they should be required.